public/assets/images/index.ts

```diff
@@ -8,5 +8,10 @@ export { default as imgLandingMic } from './img_landing_mic.svg';
 export { default as img404Background } from './img_404_background.png';
 export { default as imgReviewEmpty } from './img_review_empty.svg';
 export { default as imgMemoEmpty } from './img_memo_empty.svg';
+export { default as imgMobileBgLast } from './img_mobile_bg_last.png';
+export { default as imgMobileBgFirst } from './img_mobile_bg.svg';
```

Comment:
```suggestion
export { default as imgMobileBgLast } from './img_mobile_bg_last.png';
export { default as imgMobileBgFirst } from './img_mobile_bg_first.png';
```

![image](https://user-images.githubusercontent.com/58380158/209635520-68190926-ea41-4107-aac9-2a4928706d5d.png)

배경 이미지 png로 바꾸는 것은 어떨까요?

Reply:
처음에 png로 확인해봤는데 화질이 너무 깨져서 다른건 png로 했는데 저건 svg로 했습니다ㅠㅠ

Reply:
아하 그럼 이건 그대로 가는 걸로 합시다 !!